demo/image-segmentation/pytorch_loader.py

```diff
@@ -186,12 +185,18 @@ def __len__(self):
         return len(self.images)
 
     def __getitem__(self, idx):
-        data = {
-            "image": np.load(self.images[idx]),
-            "label": np.load(self.labels[idx])
-        }
-        data = self.rand_crop(data)
-        data = self.train_transforms(data)
+        try:
+            image = np.load(self.images[idx])
+        except:
+            image = None
+        try:
```

Comment:
Does it make sense to have labels when the image is missing or vice-versa? Otherwise code like this might be better

```
try:
  image = np.load(self.images[idx])
  label = np.load(self.labels[idx])
except:
  image = None
  label = None
```